sisyphus/prowAccessor.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sisyphus
+
+import (
+	"encoding/json"
+	"log"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	u "istio.io/test-infra/toolbox/util"
+)
+
+const (
+	lastBuildTXT = "latest-build.txt"
+	finishedJSON = "finished.json"
+	startedJSON  = "started.json"
+)
+
+// IProwAccessor defines the Prow interface
+type IProwAccessor interface {
```

Comment:
you can even call this CIinfo or something as those are not very Prow Specific. 
```goland
GetLatestRun(jobName string) (int, error)
GetResult(jobName string, runNo int) (*ProwResult, error)
Rerun(jobName string, runNo, numRerun int) error
GetRunURL() string
```

Reply:
Done